pybossa/cache/task_browse_helpers.py

```diff
@@ -298,6 +299,13 @@ def parse_tasks_browse_args(args):
         else:
             raise ValueError('invalid gold value')
 
+    in_progress = args.get('in_progress')
+    if in_progress and in_progress != 'All':
+        if in_progress in ['Yes', 'No']:
+            parsed_args['in_progress'] = in_progress
+        else:
+            raise ValueError('invalid in progress value')
```

Comment:
Would this results in a generic/cryptic message on the client that is hard to interpret?